scalyr_agent/agent_main.py

```diff
@@ -275,21 +275,24 @@ def main(self, config_file_path, command, command_options):
                 raise Exception('Caught exception when attempt to execute command %s.  Exception was %s' %
                                 (command, str(e)))
 
-    def __read_and_verify_config(self, config_file_path):
+    def __read_and_verify_config(self, config_file_path, force_http ):
```

Comment:
I don't think we need to support this from the commandline, so you can probably just get rid of this here. (by it I mean passing in the `force_http` variable)

Reply:
Done.